### ISSUE TITLE
Accessibility: WCAG 2.1 Level A: [4.1.2 Name, Role, Value]

### ISSUE THREAD

**Issue by rnunez80** [eea/volto-tabs-block#84]
Ensures interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies
Impact: serious
Violations WCAG 2.1 Level A: [4.1.2 Name, Role, Value]
(https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Comment by ichim-david** [comment]
@rnunez80 cannot reproduce using latest version, if you still encounter this issue please add a new comment with screenshot that shows the item in question failiing
